packages/cli/generators/project/templates/package.json.ejs

```diff
@@ -1,6 +1,6 @@
 {
   "name": "<%= project.name -%>",
-  "version": "1.0.0",
+  "version": "0.0.0",
```

Comment:
Maybe `0.0.1`?

Reply:
I think it would be better in `0.0.0` because Loopback is like the base of the new application that you are going to create so `0.0.1` would be Loopback with some change.

I see it that way. Then how do we do?

Reply:
There are some sections on https://semver.org/#spec-item-4:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

> How should I deal with revisions in the 0.y.z initial development phase?
The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.

We can probably starts at `0.0.1`. Just feel `0.0.0` is a bit awkward personally.

Reply:
With that document I will proceed to set the first version to `0.0.1`